src/openms/source/FORMAT/MzQCFile.cpp

```diff
@@ -54,14 +57,15 @@ using namespace std;
 
 namespace OpenMS
 { 
-
   void MzQCFile::store(const String& input_file,
                        const String& output_file,
                        const MSExperiment& exp,
                        const String& contact_name,
                        const String& contact_address,
                        const String& description,
-                       const String& label) const
+                       const String& label,
+                       const String& inputfile_feature,
+                       const String& inputfile_id) const
```

Comment:
same here. using the identifications would make it usable also for in memory data